tests/malloc/tests/01-run.py

```diff
@@ -19,7 +23,8 @@ def testfunc(child):
     for _ in range(number_of_tests):
         child.expect(r"Allocated {} Bytes at 0x[a-z0-9]+, total [a-z0-9]+\r\n"
                      .format(chunk_size))
-        child.expect(r'Allocations count: (\d+)\r\n')
+        child.expect(r'Allocations count: (\d+)\r\n',
+            timeout=ALLOCATION_TIMEOUT)
```

Comment:
Travis doesn't like this change. Better keep this on a single line (120 characters max line length is allowed):

```suggestion
        child.expect(r'Allocations count: (\d+)\r\n', timeout=ALLOCATION_TIMEOUT)
```